src/test/java/com/google/api/generator/engine/JavaCodeGeneratorTest.java

```diff
@@ -96,7 +101,7 @@
   private static final Variable bookKindVar = createVarFromVaporRef(bookKindRef, "bookKind");
 
   @Test
-  public void validJavaClass() {
+  public void validJavaClass() throws IOException {
     // Create outer class variableDecls.
```

Comment:
This IOException will be moved once DiffUtils and AssertUtils wrapper are in place.